.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,141 @@
+name: Bug report
+description: Something in an xchainjs-lib package is broken or behaves incorrectly.
+title: "[Bug] "
+labels: ["bug"]
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)